nautobot/project-static/css/base.css

```diff
@@ -791,6 +793,147 @@ td span.hover_copy:hover .hover_copy_button {
     margin-left: 2px;
 }
 
+:root {
+    --navbar-transition-duration: 350ms;
+}
+.navbar-fixed-left .navbar-nav > .dropdown > .nav-dropdown-menu > li > .buttons {
```

Comment:
Can we clean up and put these in appropriate order with the existing contents of the file? Don't want the organizational work from #5268 to go to waste. :-)

Reply:
will do!